crates/core/app.rs

```diff
@@ -1971,6 +1975,9 @@ fn flag_no_ignore_dot(args: &mut Vec<RGArg>) {
         "\
 Don't respect .ignore files.
 
+This does **not** affect whether ripgrep will ignore files and directories
```

Comment:
Just looking at other text, I think you just want `*not*` here instead of `**not**`. (This unfortunately isn't Markdown. It's AsciiDoc that gets converted into roff.)

Reply:
Oops, fixed!